esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -117,10 +108,12 @@ def _regrid_unstructured_to_rectilinear__prepare(src_mesh_cube, target_grid_cube
 
     # TODO: Record appropriate dimensions (i.e. which dimension the mesh belongs to)
 
-    grid_x, grid_y = get_xy_coords(target_grid_cube)
+    grid_x, grid_y = get_xy_dim_coords(target_grid_cube)
+    mesh = src_mesh_cube.mesh
+    assert mesh is not None
     # From src_mesh_cube, fetch the mesh, and the dimension on the cube which that
-    # mesh belongs to (mesh_dim).
-    mesh, mesh_dim = _get_mesh_and_dim(src_mesh_cube)
+    # mesh belongs to.
+    mesh_dim = src_mesh_cube.mesh_dim()
```

Comment:
Previous pseudocode version of this (L28) took `mesh` as a parameter. Is it not necessary to do this anymore?  Is there a possibility that a cube contains more than one mesh?

Reply:
The previous pseudocode version was written before this functionality was implemented in iris. We've discussed the idea of a cube containing more than one mesh and we don't anticipate this possibility. In fact I don't believe it's possible to define such a thing under the current UGRID conventions.